src/register/register-gnome/gnucash-header.c

```diff
@@ -606,6 +606,9 @@ gnc_header_init (GncHeader *header)
     header->width = 400;
     header->style = NULL;
 
+    // Set the style context for this widget so it can be easily manipulated with css
+    gtk_style_context_add_class (gtk_widget_get_style_context (GTK_WIDGET(header)), "GncRegisterHeader");
+
```

Comment:
More generally, I would use a naming convention similar to what gtk is doing. Don't add "Gnc" and use lowercase for the names.
Ideally you shouldn't even have to mention "register" because the header is a child widget of the register widget. Unfortunately that only works if you can have real css nodes, and may be more difficult to implement with css classes.